src/Iso8601.elm

```diff
@@ -341,6 +345,25 @@ iso8601 =
             )
 
 
+{-| Parse milli-, micro-, or nanoseconds and convert to milliseconds
+-}
+milliMicroNanoInMs : Parser Int
+milliMicroNanoInMs =
+    Parser.chompWhile Char.isDigit
+        |> Parser.getChompedString
+        |> Parser.andThen
```

Comment:
Using `oneOf` with multiple `paddedInt` didn't work, because `Parser.getChompedString` from the first case always commits.